### ISSUE TITLE
Git not working anymore

### ISSUE THREAD

**Issue by nickkolok** [Microsoft/vscode#42049]
- VSCode Version: code-oss-dev 1.20.0 (Commit unknown, Date unknown)</li>
- OS Version: Linux x64 4.13.0-26-generic</li>:

Steps to Reproduce:

Same as for #37981. Setting `git.path` to `/usr/bin/git` doesn't help.
Both `git status` and `/usr/bin/git status` give valid output when runned in the opened directory.

Does this issue occur when all extensions are disabled?: Yes

Console output:
```
[main 21:28:56] update#ctor - updates are disabled
[19333:0123/212901.746167:INFO:CONSOLE(231)] "%c[Extension Host] %cdebugger listening on port 5870", source: file:///media/work2/git/vscode-16-64-a/out/vs/workbench/services/extensions/electron-browser/extensionHost.js (231)
[19333:0123/212902.246448:INFO:CONSOLE(146)] "%c[Extension Host] %cdebugger inspector at chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:5870/0a0f4033-2114-4ee9-a5db-52528f89ee98
", source: file:///media/work2/git/vscode-16-64-a/out/vs/workbench/services/extensions/electron-browser/extensionHost.js (146)
[19333:0123/212902.674306:INFO:CONSOLE(89)] "%c[Extension Host] %cError: Missing actor eExtHostLogService (isMain: false)
    at RPCProtocol.assertRegistered (/media/work2/git/vscode-16-64-a/out/vs/workbench/services/extensions/node/rpcProtocol.js:60:27)
    at Object.createApiFactory (/media/work2/git/vscode-16-64-a/out/vs/workbench/api/node/extHost.api.impl.js:89:21)
    at new ExtHostExtensionService (/media/work2/git/vscode-16-64-a/out/vs/workbench/api/node/extHostExtensionService.js:153:49)
    at new ExtensionHostMain (/media/work2/git/vscode-16-64-a/out/vs/workbench/node/extensionHostMain.js:94:38)
    at /media/work2/git/vscode-16-64-a/out/vs/workbench/node/extensionHostProcess.js:98:33
    at <anonymous> %c(at /media/work2/git/vscode-16-64-a/out/vs/workbench/node/extensionHostProcess.js:101:46)", source: file:///media/work2/git/vscode-16-64-a/out/vs/base/node/console.js (89)
```

**Comment by nickkolok** [comment]
Additional info.
Commit c5ea1cf551379b5df8a64140844c4e577fa811ac
Yarn 1.3.2

**Comment by nickkolok** [comment]
Working version of `code`:

```
Version 1.19.2
Commit Unknown
Date Unknown
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

**Comment by sandy081** [comment]
Fixed by 6fe6061960dee368c052c388842666c5b53090ca
